ui/app/selectors/selectors.js

```diff
@@ -1,69 +1,19 @@
-import { NETWORK_TYPES } from '../helpers/constants/common'
 import { stripHexPrefix, addHexPrefix } from 'ethereumjs-util'
+import { createSelector } from 'reselect'
 
-
-const abi = require('human-standard-token-abi')
+import { NETWORK_TYPES } from '../helpers/constants/common'
 import {
   transactionsSelector,
 } from './transactions'
-const {
-  multiplyCurrencies,
-} = require('../helpers/utils/conversion-util')
 import {
   addressSlicer,
   checksumAddress,
 } from '../helpers/utils/util'
+import {
+  multiplyCurrencies,
+} from '../helpers/utils/conversion-util'
 
-const selectors = {
```

Comment:
Moved as `export { ... }` to the bottom as with `createSeletor` from `reselect` we are not able to use functions anymore due to `getMetaMaskAccounts` and `getMetaMaskAccountsOrdered`. We can even migrate to `export`s during `definition` if that's what you prefer. Let me know what do you think.

Reply:
@Gudahtt, sorry for some reason I missed that comment. We can keep `require` but still we need to move all to the bottom to avoid reaching a temporary dead zone, therefore, I just decided at the same time to use `ES` exports. 
In general, I agree that here it's better/easier to use individual `export`s but I just wanted to keep the changes minimal and consistent with the current approach. Anyway, if you think it's worth to migrate to individual exports let me know as it's easy to do.

Reply:
No problem - it was my bad for not making it a review comment, which GitHub is a little better at highlighting.

I don't know what you mean by "temporary dead zone", but it's not important since I agree with the move to ES6 exports regardless.

This is fine as-is! We'll be doing some bulk-changes in the near future to make our exports more consistent, so I can change this formatting then instead.